### DISCUSSION TITLE
Duplicates in vocabulary CONCEPT table

### DISCUSSION THREAD

**Discussion by smitalimaye** [community/community#121148] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I followed the steps from [ReadMe](https://github.com/paulnagy/DICOM2OMOP/blob/main/README.md) to create fhir_valuesets.csv file from part16 file. When I ran the dicom_to_omop_tables.ipynb (I did some modifications for import the  files into bigquery dataset), it populated vocabulary concept (and also vocabulary and concept_class) table. The import created some duplicates. e.g. - Tissue Velocity, Ophthalmic Axial Measurements etc

Looks like Ophthalmic Axial Measurements is related to cid-30-DICOMDevice, cid-33-Modality and cid-29-AquisitionModality (refer to [part16_fhir_valuesets.csv](https://github.com/paulnagy/DICOM2OMOP/blob/main/files/part16_fhir_valuesets.csv)). 

Should they be de-duplicated or do they need to bring this additional context (Modality, AquisitionModality and DICOMDevice) in to the concepts?

**Comment by ebndev** [comment]
Hi @smitalimaye,

Thanks for being a part of the GitHub Community, we're glad you're here!

The Profile category is for conversations related to your GitHub profile page. I have moved this post to General, however, it looks like you're looking for support from [DICOM2OMOP](https://github.com/paulnagy/DICOM2OMOP). You might want to try asking for help somewhere that focuses on this project, such as opening an issue in the project's repository or seeing if they have a support page.

It's possible that another GitHub user might have run into this same issue and can help, but we want to make sure you’re getting the best support you can and that your message goes where you intended.

Best of luck!

**Reply by smitalimaye** [reply]
Thanks, I will move it to the issue for [DICOM2OMOP](https://github.com/paulnagy/DICOM2OMOP).

**Reply by ebndev** [reply]
Hi again @smitalimaye!

If you don't mind, would you mind letting us know how you initially ended up in the [Profile category](https://github.com/orgs/community/discussions/categories/profile?discussions_q=is%3Aopen+category%3AProfile+) when creating your discussion? We are trying to investigate if there is a bug or link that may be re-directing users from the project's Discussions board, if that’s where you intended to post.

Any insight would be helpful, thanks!

**Reply by ebndev** [reply]
Never mind, found the issue! Thanks again for your time and being a part of the GitHub Community 😄
